skimage/color/colorconv.py

```diff
@@ -679,6 +679,85 @@ def rgb2xyz(rgb):
     return _convert(xyz_from_rgb, arr)
 
 
+
+def rgb2c1c2c3(rgb):
+    """RGB to C1C2C3 color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3-D array of shape ``(.., .., 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in C1C2C3 format, in a 3-D array of shape ``(.., .., 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3-D array of shape ``(.., .., 3)``.
+
+
+    References
+    ----------
+    .. [1] https://arxiv.org/abs/1702.05421
+
+    Examples
+    --------
+    >>> from skimage import data
+    >>> img = data.astronaut()
+    >>> img_c1c2c3 = rgb2c1c2c3(img)
+    """
+    out = np.arctan(rgb / np.dstack((
+        np.maximum(rgb[..., 1], rgb[..., 2]) + 0.000000001, 
+        np.maximum(rgb[..., 0], rgb[..., 2]) + 0.000000001, 
+        np.maximum(rgb[..., 0], rgb[..., 1]) + 0.000000001
+        )))
+    return out
+
+
+def rgb2maxrgb(rgb):
+    """RGB to MAXRGB color space conversion.
```

Comment:
I would welcome a short description of what the MAXRGB is... giving reference is nice but keep it easy to use :)